packages/network/avahi/package.mk

```diff
@@ -1,13 +1,14 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2009-2016 Stephan Raue (stephan@openelec.tv)
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="avahi"
-PKG_VERSION="0.7"
-PKG_SHA256="fd45480cef0559b3eab965ea3ad4fe2d7a8f27db32c851a032ee0b487c378329"
+PKG_VERSION="0.8"
+PKG_SHA256="c15e750ef7c6df595fb5f2ce10cac0fee2353649600e6919ad08ae8871e4945f"
 PKG_LICENSE="GPL"
 PKG_SITE="http://avahi.org/"
 PKG_URL="https://github.com/lathiat/avahi/archive/v$PKG_VERSION.tar.gz"
-PKG_DEPENDS_TARGET="toolchain expat libdaemon dbus connman"
+PKG_DEPENDS_TARGET="toolchain expat libdaemon dbus connman libevent gettext"
```

Comment:
do we need libevent or can you just disable it with `--disable-libevent`??

Reply:
Will do a code review and some tests.

Reply:
Hi @lrusak I have reviewed the avahi code and pull request https://github.com/lathiat/avahi/pull/186 which added libevent main loop adapter to avahi. I created a draft commit https://github.com/LibreELEC/LibreELEC.tv/commit/def3d0791fa05de4b578ad631e10fcd4c0d3fcfb and tested avahi build without libevent, but will complete a full build before the PR.

Reply:
Completed code review, testing  and raised PR #4829